Misc/NEWS.d/next/Library/2018-06-03-22-41-59.bpo-33767.2e82g3.rst

```diff
@@ -0,0 +1,2 @@
+Fix improper use of :exc:`SystemError` by :class:`mmap.mmap` objects.  Patch
```

Comment:
Could you also mention that the + and * operations will now raise `TypeError`s?

Reply:
I suggest to reword this entry something like: "```Conatenating (``+``) and repeating (``*``) of :class:`mmap.mmap` objects now raises :exc:`TypeError` instead of :exc:`SystemError`.```"